cypress/e2e/events/[eventId].cy.ts

```diff
@@ -87,7 +87,7 @@ describe('event page', () => {
     cy.login(users.testUser.email);
     cy.joinChapter(chapterId).then(() => {
       cy.rsvpToEvent({ eventId, chapterId }).then(() => {
-        cy.visit(`/events/${eventId}?ask_to_confirm=true`);
+        cy.visit(`/events/${eventId}?confirm_rsvp=true`);
 
         cy.contains('Are you sure you want to cancel your RSVP?');
         cy.findByRole('button', { name: 'Confirm' }).click();
```

Comment:
I cannot select in github whole test, but test checking if it's possible to cancel with `?confirm_rsvp=true` won't be needed with the changes.

Reply:
I see now.  Thank you!